README.md

```diff
@@ -649,7 +650,8 @@ Nim provides unique features for seamless and transparent interoperability with
 - [pararules](https://github.com/paranim/pararules) - A RETE-based rules engine made for games.
 
 
-## Development Tools
+### Development Tools
+- [moe](https://github.com/fox0430/moe) - A vim-like editor made with Nim, also supports C, Rust, Javascript, etc.
```

Comment:
It should be put under `Editor Integration` section.

Reply:
Gotcha, I'll change to that asap.